reaktive/src/jsMain/kotlin/com/badoo/reaktive/utils/ThrowableExt.kt

```diff
@@ -0,0 +1,5 @@
+package com.badoo.reaktive.utils
+
+actual fun Throwable.printStack() {
+    // no-op
```

Comment:
I had never noticed that there was no printStackTrace available for JS. Could we print the Throwable message field, then the cause field (if not null)?

Reply:
Found a way of getting stack trace for exceptions. Check :-)

Reply:
Have you tried to just print the throwable? Probably it will display interactive error object in the log: 
![image](https://user-images.githubusercontent.com/725526/58370102-7fa5b200-7efa-11e9-82b7-5ad847c79c69.png)

Reply:
If you print via `println` then it prints just class name and message. If you print via `console.log` then it provides that interactive thing. Should we use `console`? As far as I see `println` has some checks for `NodeJs` and prints either using `NodeJs` or via `console`.
![image](https://user-images.githubusercontent.com/26204457/58370340-36a32d00-7efd-11e9-8dcc-0321d256ab70.png)

Reply:
@lukaville

Reply:
@arkivanov looks like in node `console.log`/`console.error` work as expected. So I think it is ok to use `console` object:
```
➜  ~ node
> console.log(Error('test'))
Error: test
    at repl:1:13
    at Script.runInThisContext (vm.js:96:20)
    at REPLServer.defaultEval (repl.js:329:29)
    at bound (domain.js:396:14)
    at REPLServer.runBound [as eval] (domain.js:409:12)
    at REPLServer.onLine (repl.js:642:10)
    at REPLServer.emit (events.js:187:15)
    at REPLServer.EventEmitter.emit (domain.js:442:20)
    at REPLServer.Interface._onLine (readline.js:290:10)
    at REPLServer.Interface._line (readline.js:638:8)
```

Reply:
Alright will fix

Reply:
@lukaville Fixed